src/client/stylesheets/style.less

```diff
@@ -325,11 +325,26 @@ hr.wide {
 		border: none;
 		border-radius: 0;
 	  }
-  
+
 	  > a:hover,
 	  > a:focus {
 		text-decoration: none;
 		background-color:@dropdown-link-hover-bg;
 	  }
 	}
-}
\ No newline at end of file
+}
+
+@media only screen and (max-width: 600px) {
+	#searchpage-button-group {
+		display: flex;
+		flex-wrap: wrap;
+	}
+
+	#searchpage-button-group .btn {
+		width: 90%;
```

Comment:
This introduces a margin on the right.
100% looks fine to me; what was the reason for 90%?